eliot/eliot_metrics.yaml

```diff
@@ -0,0 +1,151 @@
+# statsd metrics emitted using Markus.
+#
+# When adding a new metric, make sure to add it here first.
+---
+
+eliot.pageview:
```

Comment:
With the exception of this new `eliot.pageview` metric, the rest of these were encoded in `eliot/libmarkus.py` in Python code. I just transcribed that information into YAML.